Cargo.toml

```diff
@@ -15,15 +15,14 @@ repository = "https://github.com/dusk-network/poseidon252"
 dusk-bls12_381 = {version = "0.8.0-rc.0", default-features = false}
 dusk-jubjub = {version = "0.10.0-rc.0", default-features = false}
 dusk-bytes = "0.1"
-dusk-hades = { version = "0.16.0-rc.0", default-features = false }
+dusk-hades = "0.16.0-rc.0"
 canonical = {version = "0.6", optional = true}
 canonical_derive = {version = "0.6", optional = true}
 microkelvin = {version = "0.7.1", optional = true}
 nstack = {version = "0.8", optional = true}
-dusk-plonk = {version="0.8.0-rc.1", default-features = false, optional = true}
+dusk-plonk = {version="0.8.0-rc.1", default-features = false, features = ["alloc"]}
 
 [dev-dependencies]
-canonical_host = "0.5"
```

Comment:
The removal of canonical_host should be probably belong to the previous PR or other PR related to that, I'm okay to leave as is now, but for the next time the domain of PRs should be more defined.